### PR TITLE
enabled recover mode of the lxml XMLParser.

### PR DESCRIPTION
**Pull request by hungerpirat**
[Wichtel_neu.zip](https://github.com/mrbeam/MrBeamPlugin/files/5519391/Wichtel_neu.zip)

backend does not crash anymore on the attached file.

### CODE REVIEW

**Review by amrsoll — APPROVED** [review]
Non breaking change, see documentation here:
https://lxml.de/apidoc/lxml.etree.html?highlight=xmlparser#lxml.etree.XMLParser
> recover - try hard to parse through broken XML

**Review by andywerner — APPROVED** [review]
